docs/contributing/contributing-icons.stories.mdx

```diff
@@ -0,0 +1,94 @@
+import { Meta } from '../../.storybook/components';
+
+<Meta title="Introduction/Contributing/Contributing Icons" />
+
+# Contributing Icons
+
+This page outlines the process of contributing an icon to the `@sumup/icons` package.
+
+> Note that icons should be added by internal contributors with access to the SumUp [Figma icons library](https://www.figma.com/file/vnFVuPNlqF45rkw1u9toBC/SumUp-Iconography) (internal link). If you don't have access but would like to see an icon added to `@sumup/icons`, please [open an issue](https://github.com/sumup-oss/circuit-ui/issues/new).
+
+## Adding a new icon to `@sumup/icons`
+
+1. Create a new SVG file for each icon size in [`packages/web/icons/v2/`](https://github.com/sumup-oss/circuit-ui/tree/main/packages/icons/web/v2) with the name `name_size.svg` (e.g. `add_items_24`—this will generate an `<AddItems />` component).
+2. Export the icon as SVG from the [Figma icons library](https://www.figma.com/file/vnFVuPNlqF45rkw1u9toBC/SumUp-Iconography) (internal link). If the icon isn't in the library, make a request with the design team first.
+   ![Right click on the group in Figma and choose "export as SVG"](/images/contributing-icons-export.png)
+3. Paste the SVG into your file and verify the code—refer to the "Caveats" section below.
```

Comment:
🙈
```suggestion
3. Paste the SVG into your file and verify the code — refer to the "Caveats" section below.
```

Reply:
Actually, from [a Merriam Webster article](https://www.merriam-webster.com/words-at-play/em-dash-en-dash-how-to-use):

> Spacing around an em dash varies. Most newspapers insert a space before and after the dash, and many popular magazines do the same, but most books and journals omit spacing, closing whatever comes before and after the em dash right up next to it. This website prefers the latter, its style requiring the closely held em dash in running text.

...but I don't mind, let's be newspapery and keep the spaces 😛 learned something about punctuation this morning, usually used without spaces (personal pref and probably what I'm used to from books)

Reply:
TIL! I find it easier to read with the spaces around but 🤷‍♂️